src/services/extensions/links.ts

```diff
@@ -16,14 +16,16 @@ export const ExtensionPoints = {
   MetricExploration: 'grafana-lokiexplore-app/metric-exploration/v1',
 } as const;
 
-// `plugin.addLink` requires these types; unfortunately, the correct `PluginExtensionAddedLinkConfig` type is not exported with 11.2.x
-// TODO: fix this type when we move to `@grafana/data` 11.3.x
-export const linkConfigs: Array<
+export type LinkConfigs = Array<
   {
     targets: string | string[];
     // eslint-disable-next-line deprecation/deprecation
   } & Omit<PluginExtensionLinkConfig<PluginExtensionPanelContext>, 'type' | 'extensionPointId'>
-> = [
+>;
+
+// `plugin.addLink` requires these types; unfortunately, the correct `PluginExtensionAddedLinkConfig` type is not exported with 11.2.x
+// TODO: fix this type when we move to `@grafana/data` 11.3.x
```

Comment:
AFAIK we can upgrade dependencies without needing to upgrade minimum grafana version.